cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/ExpressionHandler.kt

```diff
@@ -182,45 +182,83 @@ class ExpressionHandler(frontend: PythonLanguageFrontend) :
         return assignExpression
     }
 
+    /**
```

Comment:
Add some short example how this function models Python code -> CPG nodes.

Reply:
Done. Is this what you had in mind?

Reply:
Looks good to me. I would have added:

The code `f"{value:.2f}"` is first transformed to the equivalent expression `format(value, ".2f")` and then modeled as:

This has the `format(value, ".2f")` translation in the example, which I think is easier to read. But I'm happy with the current doc, too :)